QSB/Finder.cs

```diff
@@ -0,0 +1,163 @@
+﻿using Newtonsoft.Json;
+using QSB.Animation;
+using QSB.Events;
+using QSB.Utility;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace QSB
+{
+    public static class PlayerRegistry
```

Comment:
the filename is something else

Reply:
This class has the potential to become a "god class" - a class that does all sorts of stuff. Not a huge issue right now, but is something to look out for.

Reply:
fixed file name - keeping unresolved to remind me not to over complicate it :P